Dockerfile

```diff
@@ -25,11 +25,23 @@ WORKDIR /app
 
 ENV APP_ENV=prod
 
-RUN mkdir data var && \
+RUN mkdir var && \
     chown www-data:www-data var
 
-RUN docker-php-ext-install \
-    opcache
+RUN apk add --no-cache --virtual .build-deps $PHPIZE_DEPS && \
+    apk add --no-cache \
+        postgresql-dev \
+        postgresql-libs \
+    && \
+    docker-php-ext-install \
+        opcache \
+        pdo_pgsql \
```

Comment:
why not just install https://pkgs.alpinelinux.org/package/v3.9/community/x86_64/php7-pdo_pgsql which should not require additional libraries and/or compilation?
Same goes for https://pkgs.alpinelinux.org/package/v3.9/community/x86_64/php7-opcache

Reply:
Doesn't seem to work, get `Cannot find libpq-fe.h. Please specify correct PostgreSQL installation path`

Reply:
Looking at it, the base image doesn't use the `php7` package. `php7-*` depend on `php7-common` rather than `php7` itself, but suspect we might end up with version inconsistencies.